src/Features/CSharp/Portable/ExtractMethod/CSharpMethodExtractor.Analyzer.cs

```diff
@@ -149,7 +149,7 @@ protected override ITypeSymbol GetSymbolType(SemanticModel semanticModel, ISymbo
 
                         if (AreAllReferencesNotNull(references))
                         {
-                            return base.GetSymbolType(semanticModel, symbol).WithNullability(NullableAnnotation.NotAnnotated);
+                            return base.GetSymbolType(semanticModel, symbol).WithNullableAnnotation(NullableAnnotation.NotAnnotated);
```

Comment:
💡 It would be nice to have a comment here. I'm specifically interested to know why it's `NotAnnotated` instead of `None`.

Reply:
➡️ Ah, seems like `None` is the oblivious case, and `NotAnnotated` is the not-null case.

Reply:
Correct: this is the explicit way to say "no". Both probably would have worked, but this is more correct. Do you still feel a comment is needed?

Reply:
It took a second but I figured it out. Seems fine.